### PR TITLE
[AQ] edit pass: REST API reference docs

### PR DESCRIPTION
**Pull request by ShawnJackson**
@mikekistler, @nitinme - Here's my edit pass on the five JSON files, as requested in work item [69663](https://dev.azure.com/msft-skilling/Content/_workitems/edit/69663). Please review my changes and let me know if any of them affect technical meaning.

### CODE REVIEW

**Review by mikekistler — COMMENTED** [review]
This looks really good!

I left some comments about "optional" and "required" in descriptions and also repeating lists of allowed values. Please apply this throughout.

Tag me to re-review when those changes are done and this should be good to merge.

File: specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/openapi.json [hunk 1]
```diff
@@ -558,19 +558,19 @@
       "properties": {
         "alignMode": {
           "$ref": "#/definitions/Multivariate.AlignMode",
-          "description": "An optional field, indicating how to align different variables to the same\ntime-range. Either Inner or Outer. "
+          "description": "Optional field that indicates how to align different variables to the same\ntime range. It can be either Inner or Outer."
```
[the remark this answers is not in the document]
Comment (ShawnJackson) [reply specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/openapi.json]: Done.

File: specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/openapi.json [hunk 2]
```diff
@@ -558,19 +558,19 @@
...
         },
         "fillNAMethod": {
           "$ref": "#/definitions/Multivariate.FillNAMethod",
-          "description": "An optional field, indicating how missing values will be filled. One of\nPrevious, Subsequent, Linear, Zero, Fixed."
+          "description": "Optional field that indicates how missing values will be filled. It can be one of\nPrevious, Subsequent, Linear, Zero, or Fixed."
```
[the remark this answers is not in the document]
Comment (ShawnJackson) [reply specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/openapi.json]: Done.

File: specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/openapi.json [hunk 3]
```diff
@@ -1338,15 +1338,15 @@
...
           "type": "string",
           "format": "date-time",
-          "description": "Optional argument, timestamp of a data point (ISO8601 format)."
+          "description": "Optional argument that indicates the time stamp of a data point (ISO8601 format)."
         },
         "value": {
           "type": "number",
           "format": "float",
-          "description": "The measurement of that point, should be float."
+          "description": "Measurement of that point, which should be float."
```
[the remark this answers is not in the document]
Comment (ShawnJackson) [reply specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/openapi.json]: Done.

**Comment by ShawnJackson** [comment]
@mikekistler - I applied your suggestions globally. Please take another look.

**Review by heaths — APPROVED** [review]
A random sampling of text LGTM.

**Comment by heaths** [comment]
Be sure to go through the PoliCheck and SpellCheck failures.

The ModelValidation failures are moot since you're not changing anything besides descrptions.

For Anomaly Detector, please update the .cadl file(s) and regenerate the swagger instead of the swagger directly.

**Review by mikekistler — APPROVED** [review]
Looks good! 👍 

There is one complication with the `specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/openapi.json`. This file is actually generated using TypeSpec, so we need to make the corresponding changes to the TypeSpec source. I can do that but I'd like to separate that into it's own PR. Can you please drop that file from this PR?  I'll create a new PR to include the updated TypeSpec and your changed openapi.json.

**Comment by ShawnJackson** [comment]
> Looks good! 👍
> 
> There is one complication with the `specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/openapi.json`. This file is actually generated using TypeSpec, so we need to make the corresponding changes to the TypeSpec source. I can do that but I'd like to separate that into it's own PR. Can you please drop that file from this PR? I'll create a new PR to include the updated TypeSpec and your changed openapi.json.

@mikekistler - I dropped openapi.json from this PR.

**Comment by heaths** [comment]
The spell check issue was fixed in #23379 and the model validation issues weren't introduced here.

**Pull request merged**